Builds/XCode/README.md

```diff
@@ -164,6 +169,12 @@ scons
 
 See: [here](https://ripple.com/wiki/Rippled_build_instructions#Building)
 
+## Unit Tests (Recommended)
+
+rippled builds a set of unittest into the server executable. To run these unit
```

Comment:
Ultra-minor nit: consider, for this usage, "set of unittest" -> "set of unit tests"